contrib/linearize/linearize-data.py

```diff
@@ -170,7 +170,7 @@ def writeBlock(self, inhdr, blk_hdr, rawblock):
 			self.highTS = blkTS
 
 		if (self.blkCountOut % 1000) == 0:
-			print('%i blocks scanned, %i blocks written (of %i, %.1f%% complete)' % 
+			print('%i blocks scanned, %i blocks written (of %i, %.1f%% complete)' %
```

Comment:
Looks like the travis linter fails here, because tabs are used instead of spaces for indenting.
Which was already the case.
To match the style of the other files, I think you'd need to change tabs to four spaces throughout the file.